### ISSUE TITLE
Misplaced accessibilityAPI + accessibilityControl properties

### ISSUE THREAD

**Issue by LaurenceZaysser** [w3c/a11y-discov-vocab#149]
This is just a small delail in the document's form.

Two Schema.org accessibility properties have been misplaced: _accessibilityAPI_, and _accessibilityControl_. 

As a matter of fact, they have been placed under _accessibilityHazard_ chapter, instead of being separate chapters = TOC entries. As a consequence, when you parse the TOC, you do not see them, and think they are missing.

Even if the information attached to _accessibilityAPI_, and _accessibilityControl_ properties is very short, having a fully logical and hierarchical TOC would help a lot everybody reading the spec. ;-)

Thanks!

**Comment by clapierre** [comment]
fixed placement move to bottom of list.
